src/containers/Welcome/Step1/Step1.tsx

```diff
@@ -0,0 +1,335 @@
+};
+
+const Step1 = (p: Wizard.StepProps) => {
+  // Refs
+  const resetExecuted = useRef(false);
+
+  // Hooks
+  const { Portal } = usePortal({
+    bindTo: document && document.getElementById('wizard-buttons') as HTMLDivElement,
+  });
+  const { t, i18n } = useTranslation();
+  const { state, actions } = useStateMachine({ update: updateAction(p.storeKey), reset: resetStore() });
+  const {
+    setType, setDoGoBack, setLogoSize,
+  } = useHeaderContext();
+  const history = useHistory();
+  const store = state?.[p.storeKey];
+  const {
+    control,
+    formState,
+    handleSubmit,
+    watch,
+    setValue,
+    reset,
+  } = useForm({
+    defaultValues: store,
+    resolver: yupResolver(schema),
+    mode: 'onChange',
+  });
+  const { isValid } = formState;
+  const lang = watch('language');
+
+  // States
+  const [activeStep, setActiveStep] = useState(true);
+  const [supportedLang, setSupportedLang] = useState<{ value: string; label: string; }[]>([]);
+  const [ipLimit, setIpLimit] = useState(false);
+
+  // Callbacks
+  const onSubmit = async (values: Step1Type) => {
+    if (values) {
+      actions.update(values);
+      if (p.nextStep) {
+        setActiveStep(false);
+        history.push(p.nextStep);
+      }
+    }
+  };
+
+  /**
+   * The function `getOptionsCountry` filters and formats country data based on supported countries and
+   * returns the formatted options.
+   * @returns An array of objects with the properties from the `supportedCountriesOp` array, where each
+   * object also has a `label` property that is derived from the translation key
+   * `main:countries.${current.value}`.
+   */
+  const getOptionsCountry = () => {
+    const supportedCountriesOp = countryData.filter(op => supportedCountries.includes(op.value));
+    const formattedOptions = supportedCountriesOp.reduce((acc: CountryDataProps[], current) => {
+      acc.push({ ...current, label: t(`main:countries.${current.value}`) });
+      return acc;
+    }, []);
+    return formattedOptions;
+  };
+
+  // Effects
+  /* This `useEffect` hook is responsible for resetting the form values when the component mounts or
+  when the `store` state changes. Here's a breakdown of what it does: */
+  useEffect(() => {
+    if (resetExecuted.current) {
+      resetExecuted.current = false;
+      reset(store);
+    }
+  }, [store, reset]);
+
+  useEffect(() => {
+    scrollToTop();
+    // Hide back arrow in header if neccesary
+    setDoGoBack(null);
+    setLogoSize('regular');
+    setType('null');
+  }, [setDoGoBack, setLogoSize, setType]);
+
+  useEffect(() => {
+    i18n.changeLanguage(lang);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [i18n, lang]);
+
+  /* This `useEffect` hook is responsible for initializing the form values for the country and language
+  fields when the component mounts. Here's a breakdown of what it does: */
+  useEffect(() => {
+    const localStorageCountry = localStorage.getItem('countryResult');
```

Comment:
Change this `countryResult` to `virumap-countryResult` also. If you used it previously, maybe you have more languages than are really supported